core/templates/pages/welcome-guide-page/welcome-guide-page.module.ts

```diff
@@ -0,0 +1,93 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Module for the welcome guide page.
+ */
+
+import 'core-js/es7/reflect';
+import 'zone.js';
+
+import { BrowserModule } from '@angular/platform-browser';
+import { Component, NgModule, StaticProvider } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { HttpClientModule, HTTP_INTERCEPTORS } from '@angular/common/http';
+import { RequestInterceptor } from 'services/request-interceptor.service';
+
+// This component is needed to force-bootstrap Angular at the beginning of the
+// app.
+@Component({
+  selector: 'service-bootstrap',
+  template: ''
+})
+export class ServiceBootstrapComponent {}
+
+import { AppConstants } from 'app.constants';
+import { InteractionsExtensionsConstants } from
+  'interactions/interactions-extension.constants';
+import { ObjectsDomainConstants } from
+  'domain/objects/objects-domain.constants';
+
+@NgModule({
+  imports: [
+    BrowserModule,
+    HttpClientModule
+  ],
+  declarations: [
+    ServiceBootstrapComponent
+  ],
+  entryComponents: [
+    ServiceBootstrapComponent
+  ],
+  providers: [
+    AppConstants,
+    InteractionsExtensionsConstants,
+    ObjectsDomainConstants,
+    {
+      provide: HTTP_INTERCEPTORS,
+      useClass: RequestInterceptor,
+      multi: true
+    }
+  ]
+})
+class WelcomeGuidePageModule {
+  // Empty placeholder method to satisfy the `Compiler`.
+  ngDoBootstrap() {}
+}
+
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { downgradeModule } from '@angular/upgrade/static';
+
+const bootstrapFn = (extraProviders: StaticProvider[]) => {
+  const platformRef = platformBrowserDynamic(extraProviders);
+  return platformRef.bootstrapModule(WelcomeGuidePageModule);
+};
+const downgradedModule = downgradeModule(bootstrapFn);
+
+declare var angular: ng.IAngularStatic;
+
+angular.module('oppia', [
+  'dndLists', 'headroom', 'infinite-scroll', 'ngAnimate',
+  'ngAudio', require('angular-cookies'), 'ngImgCrop', 'ngJoyRide', 'ngMaterial',
+  'ngSanitize', 'ngTouch', 'pascalprecht.translate',
+  'toastr', 'ui.bootstrap', 'ui.sortable', 'ui.tree', 'ui.validate',
+  downgradedModule
```

Comment:
Could you please set these according to https://github.com/oppia/oppia/blob/c513a4083d7043083501d91e12e4d0f0681186ae/core/templates/pages/pending-account-deletion-page/pending-account-deletion-page.module.ts#L76?

Reply:
Done!